FlorenceBML/BMLor403/BMLor403.xml

```diff
@@ -34,8 +35,421 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                <locus from="175r" to="200r"></locus>
+                                <title ref="LIT3578Prayer"/>
+                                <incipit xml:lang="gez">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ። ጸሎቱ፡ ለብፁዕ፡ ወለቅዱስ፡ ቆጵርያኖስ፡ ጸሎቱ፡ ትብጽሐነ፡ አሜን። ዛቲ፡ ጸሎት፡ ጽንዕ፡ 
+                                    ለነገሥት፡ ወስደት፡ ለሰይጣናት፡ ዓይን፡ እኩይ፡ ወራእዩ፡ እኩይ። ወፈታሒ፡ እሱራን፡ ውስተ፡ አሕላም፡ እኩያን።</incipit>
+                                <explicit xml:lang="gez">ተማኅፀንኩ፡ በእግዚአ፡ ሚካኤል፡ ወገብርኤል። እግዚአ። ሱሩፌል፡ ወኪሩቤል። ወበእግዚአ። ሩፋኤል። ኡራኤል። ከመ፡ ኢይቀርባኒ፡ ሥራይ። 
+                                    ወአትናአር። ማያኤል። ጣያአል። ባዝንቱ፡ ቃል፡ ተማኅፀንኩ፡ አነ፡ ገብርከ፡ <gap reason="illegible"/>ለዓለመ፡ ዓለም። አሜን። በአብ፡ በማሩኤል። በወልድ፡ በምናቴር። በመንፈስ። በምናቴር። 
+                                    በመንፈስ፡ ቅዱስ፡ አብያቴር። ኣድኅን። አምላከ፡ እግዚአብሔር፡ አብ፡ ክርስቶስ፡ ወልድ፡ ሣልሣዊ፡ መንፈስ፡ ቅዱስ። ያስሬ፡ ሕፀት። ካርካር። ትዓት፡ የሰር። ጕልድም። አድኅን።
+                                    ኣምላከ፡ አቡከ፡ ዮቅ። ወእመከ፡ ስሕ።</explicit>
+                                <note>The name of the owner <ref target="PRS0000Simon">ስምዖን</ref> has been inscribed various times on 
+                                    <locus target="#178v #179r #179v #181r #190r #196r #196v #197r #198v #200r"/>, but has been erased throughout.</note>
+                            </msItem>
+                            <msItem xml:id="ms_i10">
+                                <locus from="200v" to="207v"/>
+                                <title ref="LIT6730PrayerExorcism"/>
+                                <incipit xml:lang="gez">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ጸሎት፡ በዘ፡ ይስዓር፡ ኵሉ፡ መናግንቲሁ፡ ለሰይጣን፡ ወይሰድድ፡ ኵሉ፡ ደዌ፡ ወሕማም፡ 
+                                    ወኵሉ፡ መናፍስተ፡ ርኩሳን፡ አምላዕለ፡ ነፍሱ፡ ወሥጋሁ፡ ለገብርከ። <gap reason="illegible"/> እምላዕለ፡ ዘይዓንቆ፡ ወይጸውሮ፡ ለዝንቱ፡ መጽሐፍ፡ ወእምዝ፡
+                                    ጸለዩ፡ ወአንባቡ፡ ሎቱ፡ በዝ፡ መጽሐፍ፡ በኀይለ፡ መስቀሉ፡ ለክርስቶስ፡ ኢይምጻእ፡ ደዌ፡ ወሕማም፡ ለዘ፡ የዓንቆ፡ ወይጸውሮ፡ ለዝንቱ፡ መጽሐፍ፡ አድኅኖ። ለገብርከ።</incipit>
+                                <explicit xml:lang="gez">ሊተ፡ ለገብረ፡ <gap reason="illegible"/> እግዚአብሔር፡ ወኵሎ፡ ዘዚአየ፡ ወለእገሊሂ፡ ዘአነ፡ ጸለይኩ፡ ሎቱ፡ በዝ፡ መጽሐፍ፡ 
+                                    ለእመሂ፡ ብእሲ፡ ወለእመሂ፡ ብእሲት፡ ወእመሂ፡ ዲበ፡ ማይ፡ ወእመሂ፡ ተሐጽበ፡ ወእመሂ፡ ረቀይክዎ፡ ወሰትየ፡ ይድኀን፡ እምኵሉ፡ እኩይ፡ ኢትክሉ፡ አሕሲመ፡ ላዕሌየ፡ 
+                                    ከመዝ፡ በዝ፡ መሐላ፡ አምሐልኩክሙ፡ ከመ፡ ትፃኡ፡ ወትሰደድ፡ እምኀቤነ፡ ወኵሉ፡ ዘጸሮ፡ ወዓነቆ፡ ለዝ፡ መጽሐፍ፡ የሐዩ፡ አንተ፡ እግዚኦ፡ ዕቀባነ፡ ወተማኅፀነነ፡ እምኵሉ፡
+                                    ዘኢንፈቅድ፡ ወሳውረነ፡ ባጽላሎተ፡ ክነፈከ፡ ክድነነ፡ እምኩሉ፡ ዘንትገሐስ፡ ወኩነኒ፡ መድኀኒተ፡ ነፍስየ፡ ወሥጋየ፡ ወስረይ፡ ሊተ፡ ኵሉ፡ ኃጢአትየ፡ ሊተ፡ ለገብርከ፡ ለዓለመ፡ 
+                                    ዓለም፡ አሜን።</explicit>
+                                <note>The name of the owner <ref target="PRS0000Simon">ስምዖን</ref> was inscribed on <locus target="#200v #201r #204r"/>, 
+                                    but has been erased.</note>
+                            </msItem>
+                            <msItem xml:id="ms_i11">
+                                <locus from="208r" to="209v"/>
+                                <title ref="LIT0000">Nicene creed</title>
```

Comment:
LIT3308Nicene?

Reply:
Yes. Sorry, I forgot to look up the narrative ID in the end of my working process. Done it by now.